python/ray/autoscaler/_private/kuberay/run_autoscaler.py

```diff
@@ -51,17 +63,29 @@ def run_kuberay_autoscaler(cluster_name: str, cluster_namespace: str):
         cluster_name, cluster_namespace
     )
 
-    Monitor(
-        address=ray_address,
-        # The `autoscaling_config` arg can be a dict or a `Callable: () -> dict`.
-        # In this case, it's a callable.
-        autoscaling_config=autoscaling_config_producer,
-        monitor_ip=head_ip,
-        # Let the autoscaler process exit after it hits 5 exceptions.
-        # (See ray.autoscaler._private.constants.AUTOSCALER_MAX_NUM_FAILURES.)
-        # Kubernetes will then restart the autoscaler container.
-        retry_on_failure=False,
-    ).run()
+    # Init GCS client
+    _initialize_internal_kv(GcsClient(ray_address))
+    if is_autoscaler_v2(fetch_from_server=True):
+        from ray.autoscaler.v2.monitor import AutoscalerMonitor as MonitorV2
+
+        MonitorV2(
+            gcs_address=ray_address,
```

Comment:
Why do we use `address` in `Monitor` but `gcs_address` in `MonitorV2`?

Reply:
ah, we should keep it sync.